internal/deviations/deviations.go

```diff
@@ -575,8 +570,6 @@ var (
 
 	bgpMD5RequiresReset = flag.Bool("deviation_bgp_md5_requires_reset", false, "Device requires a BGP session reset to utilize a new MD5 key")
 
-	qosDroppedOctets = flag.Bool("deviation_qos_dropped_octets", false, "Set to true to skip checking QOS Dropped octets stats for interface")
```

Comment:
This deviation is currently used by other vendors too, so I dont think we can remove them.

Reply:
This code is in cisco specific config function will not affect other vendors.I have added deviations in deviations folder